README.md

```diff
@@ -96,61 +93,67 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
 
 To start using the SDK in your applications, create a Client instance and then invoke the API on it:
 
-#### With login ID and password
+#### Define connection parameters
 
-```python3
-#!/usr/bin/env python3
+In order to login to conjur you need to have 5 parameters known from advance.
 
-from conjur import Client
+```
+conjur_url = "https://my_conjur.com"
+account = "my_account"
+username = "user1"
+password = "SomeStr@ngPassword!1"
+ssl_verification_mode = SslVerificationMode.TRUST_STORE
```

Comment:
how will they know what to put into this one? is it user defined? where will they get this from? The others are obvious. But is this one?

Reply:
We explain it later